### DISCUSSION TITLE
No pipeline packages found in your current environment.

### DISCUSSION THREAD

**Discussion by poethan** [explosion/spaCy#8416] [Help: Installation] [answered]
I use MacBook Air to install spaCy. the installation was correct, then the validation comment says : No pipeline packages found in your current environment.
what does this mean? and what shall I do?
details bellow:
Lifengs-Air:~ lifenghan$ pip install -U spacy
Requirement already satisfied: spacy in /anaconda3/lib/python3.7/site-packages (3.0.6)
Requirement already satisfied: blis<0.8.0,>=0.4.0 in /anaconda3/lib/python3.7/site-packages (from spacy) (0.7.4)
Requirement already satisfied: pydantic<1.8.0,>=1.7.1 in /anaconda3/lib/python3.7/site-packages (from spacy) (1.7.4)
Requirement already satisfied: catalogue<2.1.0,>=2.0.3 in /anaconda3/lib/python3.7/site-packages (from spacy) (2.0.4)
Requirement already satisfied: thinc<8.1.0,>=8.0.3 in /anaconda3/lib/python3.7/site-packages (from spacy) (8.0.5)
Requirement already satisfied: srsly<3.0.0,>=2.4.1 in /anaconda3/lib/python3.7/site-packages (from spacy) (2.4.1)
Requirement already satisfied: packaging>=20.0 in /anaconda3/lib/python3.7/site-packages (from spacy) (20.9)
Requirement already satisfied: setuptools in /anaconda3/lib/python3.7/site-packages (from spacy) (57.0.0)
Requirement already satisfied: pathy>=0.3.5 in /anaconda3/lib/python3.7/site-packages (from spacy) (0.5.2)
Requirement already satisfied: numpy>=1.15.0 in /anaconda3/lib/python3.7/site-packages (from spacy) (1.15.4)
Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /anaconda3/lib/python3.7/site-packages (from spacy) (3.0.5)
Requirement already satisfied: typer<0.4.0,>=0.3.0 in /anaconda3/lib/python3.7/site-packages (from spacy) (0.3.2)
Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /anaconda3/lib/python3.7/site-packages (from spacy) (1.0.5)
Requirement already satisfied: requests<3.0.0,>=2.13.0 in /anaconda3/lib/python3.7/site-packages (from spacy) (2.21.0)
Requirement already satisfied: wasabi<1.1.0,>=0.8.1 in /anaconda3/lib/python3.7/site-packages (from spacy) (0.8.2)
Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /anaconda3/lib/python3.7/site-packages (from spacy) (2.0.5)
Requirement already satisfied: typing-extensions<4.0.0.0,>=3.7.4 in /anaconda3/lib/python3.7/site-packages (from spacy) (3.10.0.0)
Requirement already satisfied: jinja2 in /anaconda3/lib/python3.7/site-packages (from spacy) (2.10)
Requirement already satisfied: tqdm<5.0.0,>=4.38.0 in /anaconda3/lib/python3.7/site-packages (from spacy) (4.61.1)
Requirement already satisfied: spacy-legacy<3.1.0,>=3.0.4 in /anaconda3/lib/python3.7/site-packages (from spacy) (3.0.6)
Requirement already satisfied: zipp>=0.5 in /anaconda3/lib/python3.7/site-packages (from catalogue<2.1.0,>=2.0.3->spacy) (3.4.1)
Requirement already satisfied: pyparsing>=2.0.2 in /anaconda3/lib/python3.7/site-packages (from packaging>=20.0->spacy) (2.3.0)
Requirement already satisfied: smart-open<4.0.0,>=2.2.0 in /anaconda3/lib/python3.7/site-packages (from pathy>=0.3.5->spacy) (3.0.0)
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy) (3.0.4)
Requirement already satisfied: urllib3<1.25,>=1.21.1 in /anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy) (1.24.1)
Requirement already satisfied: certifi>=2017.4.17 in /anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy) (2018.11.29)
Requirement already satisfied: idna<2.9,>=2.5 in /anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy) (2.8)
Requirement already satisfied: click<7.2.0,>=7.1.1 in /anaconda3/lib/python3.7/site-packages (from typer<0.4.0,>=0.3.0->spacy) (7.1.2)
Requirement already satisfied: MarkupSafe>=0.23 in /anaconda3/lib/python3.7/site-packages (from jinja2->spacy) (1.1.0)
Lifengs-Air:~ lifenghan$ python -m spacy validate
✔ Loaded compatibility table

================= Installed pipeline packages (spaCy v3.0.6) =================
ℹ spaCy installation: /anaconda3/lib/python3.7/site-packages/spacy

No pipeline packages found in your current environment.

**Comment by Pandalei97** [comment, marked as answer]
Hi, `pip install -U spacy` doesn't download the pipeline packages. Use `python -m spacy download` to get the pretrained pipeline that you need. More details in https://spacy.io/models
